src/ParseUser.ts

```diff
@@ -812,11 +812,13 @@ class ParseUser<T extends Attributes = Attributes> extends ParseObject<T> {
    * <code>current</code> will return <code>null</code>.
    *
    * @param {object} options
+   * @param {boolean} [options.clearSession] If true, the session token will be
```

Comment:
I think this needs a better explanation. It's now a rather technical description but not so clear what the purpose of the option is and why / when one should use it.

Reply:
What do you recommend we change it to?

Reply:
I don't have a suggestion because I don't understand what the option exactly is for, from a practical perspective.